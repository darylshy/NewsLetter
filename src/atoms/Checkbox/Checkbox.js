import React from "react";

const Checkbox = props => {
  return <input type="checkbox" />;
};

export default Checkbox;
