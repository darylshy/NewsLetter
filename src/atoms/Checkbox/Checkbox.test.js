import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Checkbox from './Checkbox';
import React from 'react';

configure({ adapter: new Adapter() });

describe('<Checkbox />', () => {
  let component;
  beforeEach(() => {
    const mock = jest.fn();
    component = shallow(<Checkbox handleChecked={mock} />);
  });
  it('should render an Checkbox component', () => {
    expect(component).toMatchSnapshot();
  });
  it('should find a Checkbox', () => {
    expect(component.find('input')).toHaveLength(1);
  });
});
