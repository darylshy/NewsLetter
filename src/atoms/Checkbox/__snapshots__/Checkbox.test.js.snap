// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkbox /> should render an Checkbox component 1`] = `ShallowWrapper {}`;
